README.md

```diff
@@ -28,16 +28,27 @@ Usage
         console.log(shown.length + ' objects shown.');
     });
 
+    // remove the spatial index when the layer is unused
+    map.unindexLayer(layer);
 ```
 
+`map.unindexLayer()` function accepts an `options` object as an optional second parameter to defined the bounds of the layer. It could accept:
```

Comment:
typo: to define~~d~~

I would say something like: *the bounds to be removed from the index. It should have one of the following attributes:*